.github/release-drafter-python.yml

```diff
@@ -1,9 +1,5 @@
 _extends: kaskada:.github/release-drafter.yml
 
-name-template: Python $RESOLVED_VERSION
-tag-template: python@v$RESOLVED_VERSION
-tag-prefix: python@v
-
-# Only include PRs with one of these labels
-include-labels:
-  - python
\ No newline at end of file
+name-template: Kaskada $RESOLVED_VERSION-a.0
+tag-template: v$RESOLVED_VERSION-a.0
+tag-prefix: v
```

Comment:
Since there are no `include-labels` specified, this will include all PRs in the repo.